### ISSUE TITLE
A11y_Windows Terminal_Windows PowerShell_Reflow: When reflow is performed, user is not able to view complete Tab name

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/terminal#5418]
GitHubTags: #A11y_WindowsTerminal; #WindowsTerminal; #A11yTCS; #A11yWCAG2.1; #Win10; #DesktopApp; #Reflow; #MAS1.4.10; #C+E-H; #A11ySev2;

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-tagott%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Windows Terminal
Version: 0.10.781.0
OS: Windows 10 (Version 1909, Build: 18363.778)

**Note:** Similar issue is observed in 'About' page. Not able to view complete content (Links)

**Repro Steps:**
1.Go to settings>System>Display resolution (1280 X 1024)>Advanced Scaling settings>Scaling size 400>Apply and Signout
2.Hit the URL "aka.ms/terminal" and install "Windows terminal (Preview)" application
3.Open "Windows terminal (Preview)" application
4.Verify whether the "Tab" name is completely visible or not

**Actual Result:**
When reflow is performed, Tab name is not clearly visible.

**Expected Result:**
When reflow is performed, Tab name should be clearly visible to the user.

**User Impact:**
Low vision user will find difficulty, if the content is not completely visible when reflow is performed.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=5abfc401-5e54-4a1e-ab3b-5cb7f35ca51a

![Bug_When reflow is performed, 'Tab' name is not visible](https://user-images.githubusercontent.com/51435266/79730869-1ba88b00-830f-11ea-99a2-823444cf6671.png)

![Bug 5418_About_Note section](https://user-images.githubusercontent.com/51435266/79951178-dbbfe000-8495-11ea-8b49-d7603b933c6f.png)

**Comment by carlos-zamora** [comment]
I tried adding `MinWidth="385"` to TabRowControl.xaml, but I didn't notice a difference.

**Comment by carlos-zamora** [comment]
A minimum height was originally added to allow the About menu to be seen. This minimum height was removed from the referenced PR because we don't foresee a circumstance where your Terminal is in that state, and you can't just resize it to make the content fit properly.

**Comment by carlos-zamora** [comment]
@Amulya-95  are you still seeing this issue, or can we close it?

**Comment by Amulya-95** [comment]
@Alexandria-R
The issue is resolved for 'Tab', where user is able to view Tab name when reflow is performed.

![Fixed Bug 5418](https://user-images.githubusercontent.com/51435266/82901258-555a5a80-9f7b-11ea-98a2-c9f034aab124.png)

But, in 'About' dialog links are not visible. As per the above comments from Product team, can I close the bug?

Please find the attachment for reference.
![Not fixed Bug 5418](https://user-images.githubusercontent.com/51435266/82902120-9c951b00-9f7c-11ea-9be0-f3a74d17e186.png)

**Comment by Alexandria-R** [comment]
I don't believe this can be closed, as the loss of links seems quite relevant.

**Comment by DHowett** [comment]
It is an explicit user choice to make the window so small that its contents cannot be displayed, and it has a very easy workaround: make the window bigger. Terminal's default configuration does not automatically make the window unreadably small.

When I run a web browser and make the window so small that there is no content area, then I open the "settings" page, I see the same behavior:

![image](https://user-images.githubusercontent.com/189190/84546930-4c75c280-acb7-11ea-9d56-428e9710e4db.png)

**Comment by Alexandria-R** [comment]
Please attend Office Hours to discuss this bug
